### ISSUE TITLE
About model training

### ISSUE THREAD

**Issue by xinyuz8270** [openai/improved-diffusion#117]
Do I need to take any other steps during the training process? After training for a few days, the loss did not continue to decrease and the sampling results obtained were also very blurry. Do anyone have any training suggestions?
Here are some of my sampling results. I trained it for 250K iterations.
![real_data](https://github.com/openai/improved-diffusion/assets/96166705/2137f7da-a5c6-483e-8102-066517916108)

**Comment by cuonghoang29** [comment]
Hi there. Did you solve your problem? I had the same issue. How many images do u have for the training?

**Comment by 1zeryu** [comment]
I had the same issue. How many images do u have for the training?

**Comment by cuonghoang29** [comment]
My dataset is only 1000 images, but after checking the code, I found that we should use **random crop** instead of using center crop (because only 1k images). In my case, If I train from scratch (4 GPUs A100), after 100k steps the generated images will be clearer. If I use a pre-trained model to train, after about 50k steps the generated would be better.

**Comment by xinyuz8270** [comment]
> Hi there. Did you solve your problem? I had the same issue. How many images do u have for the training?

Sorry for not replying in time. The ImageNet dataset I used for training has about 1.2 million images. After checking my code, I found that my problem was caused by a coding error in the sampling process. So I'm sorry I can't answer your question.
